main_parm:
  robot_select: MINI_CHEETAH
  running_type: simulation
  